plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePageSink.java

```diff
@@ -162,6 +165,7 @@ public HivePageSink(
 
         this.session = requireNonNull(session, "session is null");
         this.targetMaxFileSize = Optional.ofNullable(HiveSessionProperties.getTargetMaxFileSize(session)).stream().mapToLong(DataSize::toBytes).findAny();
+        this.isTransactional = isTransactional;
```

Comment:
move before `this.hdfsEnvironment = ...`

(this isn't ideal; ideally fields, constructor params and assignments follow same ordering, but this is a mess here, so ideal place doesn't exist)